collatz-conjecture/collatz_conjecture.py

```diff
@@ -1,2 +1,11 @@
 def steps(number):
```

Comment:
Add a docstring to the function (describe what the inputs are and what the function returns and a general idea of what the function does).